srcpkgs/conduit/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'conduit'
+pkgname=conduit
+version=0.3.0
+revision=1
+wrksrc="conduit-v${version}"
+build_style=cargo
+build_helper=rust
```

Comment:
why is the build_helper included here? The cargo build_style already does that.

Reply:
Oops, that was leftover from when I was experimenting with how the `cargo` build_style and `rust` build_helper worked.